ranger-plugin/integration/service/src/test/java/org/apache/hawq/ranger/integration/service/tests/common/ServiceTestBase.java

```diff
@@ -67,7 +67,8 @@
     private static final String RANGER_URL = String.format("http://%s:%s/service/public/v2/api", RANGER_HOST, RANGER_PORT);
     private static final String RANGER_POLICY_URL = RANGER_URL + "/policy";
 
-    private static final int POLICY_REFRESH_INTERVAL = 6000;
+    private static final String POLICY_WAIT_INTERVAL_PROP_NAME = "policy.wait.interval.ms";
```

Comment:
ranger.plugin.hawq.policy.pollIntervalMs is a real property used by the service and determines the frequency of policy refreshes performed by Ranger Plugin.

policy.wait.interval.ms is the property used only by integration test suite to wait between updating policy in Ranger and trying to evaluate the request. In our tests we currently set wait.interval = 3 * pollInterval, even that sometimes causes random test failures when the system is under heavy load. We can optimize this further, if needed.